src/Microsoft.OData.Edm/Csdl/Serialization/EdmModelReferenceElementsJsonVisitor.Async.cs

```diff
@@ -0,0 +1,143 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="EdmModelReferenceElementsJsonVisitor.Async.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+#if NETSTANDARD2_0
+using Microsoft.OData.Edm.Vocabularies;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace Microsoft.OData.Edm.Csdl.Serialization
+{
+    /// <summary>
+    /// The visitor for outputting Edm reference elements for referenced model.
+    /// </summary>
+    internal partial class EdmModelReferenceElementsJsonVisitor

```

Comment:
Across this file: Confirm if constants for `$Include`, `$Reference` exist and use that if possible

Reply:
The constants do not exist, they have hardcoded that way through out. Here are some of the places used:
- https://github.com/OData/odata.net/blob/release-7.x/src/Microsoft.OData.Edm/Csdl/Parsing/CsdlJsonParser.cs
- https://github.com/OData/odata.net/blob/release-7.x/src/Microsoft.OData.Edm/Csdl/Serialization/EdmModelReferenceElementsJsonVisitor.cs